src/unix/linux_like/linux/mod.rs

```diff
@@ -1036,6 +1036,12 @@ s! {
         pub get_args: __u16,
         pub name: [c_char; ::IFNAMSIZ],
     }
+    pub struct epoll_params {
+        busy_poll_usecs: __u32,
+        busy_poll_budget: __u16,
+        prefer_busy_poll: __u8,
+        __pad: __u8,
+    }
```

Comment:
The source seems to use uint32_t/uint16_t/uint8_t so these could probably just be u32/u16/u8. I think we use the dunder integers for the others just to be consistent.

Also, any reason these aren't pub?

Reply:
Sure, my bad. I will export all but `__pad` and use the Rust integers.